packages/frontend/src/ee/features/embed/EmbedDashboard/components/EmbedMarkdownTile.tsx

```diff
@@ -12,14 +12,14 @@ export const EmbedMarkdownTile: React.FC<
     const { languageMap } = useEmbed();
 
     const translatedTile = useMemo(() => {
-        if (!languageMap) return props.tile;
+        if (!languageMap || !props.tile) return props.tile;
+
+        const properties =
+            languageMap.dashboard?.[dashboardSlug]?.tiles?.[tileIndex]
```

Comment:
Fixes this bug https://github.com/lightdash/lightdash/issues/16328

This seems to be happening because my translation.json is incorrectly mapped, so some properties can't be found in my dashboard.